packages/core/src/css/core.scss

```diff
@@ -55,7 +55,7 @@
 
   // Typography tokens
   // --------------------------------------------------
-  --way-font-family: -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
+  --way-font-family: 'Ubuntu', -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
```

Comment:
**issue:** Nee aub niet. Ik wil gewoon system fonts default. Als iemand een custom font wilt gebruiken dan kan dat in het project zelf via een `--way-font-family` override.